website/docs/moderation/notes.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Notes
+---
+
+import { AvailablePermissionsDisplay } from '../../src/components/AvailablePermissionsDisplay';
+
+> A system bringing staff notes to your server.
+
+## The Problem
+
+As it stands, there is no good way to share notes about users across server staff and the one Discord provides is heavily limited:
+
+- Notes can only be set on a per-user basis
+- Notes can only be 256 characters long
+- You cannot set multiple notes easily
+
+On larger servers however, this may become an issue. This custom command aims to solve this problem.
+It provides every functionality a server may need, that is:
+
+- Server staff can set up to ten different notes on one individual user, maxing out at 500 characters each
+- Server staff can easily view notes of each user, as well as delete them when necessary
+- Server administrators can purge the entire system if need be
+
+#### Trigger
+
+**Type**: RegEx <br />
+**Trigger**: `\A(?:\-|<@!?204255221017214977>)\s*notes?(?: +|\z)`
+**Case-Sensitive**: false
+
+:::caution
+
+Make sure to replace `-` with your prefix, should it be different.
+
+:::
+
+## Usage
+
+- `notes help`: List all subcommands
+- `notes set <user> <text>`: set a note on user
+- `notes get <user>`: get all user's notes
+- `notes del <user> <id>`: delete given note on user
+- `notes delall <user>`: delete all notes on user
+- `notes nuke`: delete all ever recorded notes
+
+## Configuration
+
+<AvailablePermissionsDisplay />
+
+- 📌 $BASE_PERMISSION <br />
+This permission should be the permission which all server staff have - on most servers, this is `ManageMessages`. If you want to lock this system behind a more exclusive permission, please view the full list under `$NUKE_PERMISSION` - it lists all of them.
+
+- 📌 $NUKE_PERMISSION <br />
+  This permission is quite a dangerous one: It grants the ability to purge the entire system. Most servers consider users with `ManageServer` permission as an admin, hence this is the default. It is recommended to change this to a more exclusive permission rather than a common one. `Adminstrator` would be a fitting candidate, as it's a quite rarely given permission, `ManageRoles` however may not.
+
+- 📌 $DELETE*TIMEOUT <br />
```

Comment:
Somehow this displays as a `*`, but it's an underscore as intended in my editor >
![image](https://user-images.githubusercontent.com/71897876/176767943-716639c6-bc7b-47b9-8f9a-793a9f8a95a0.png)